cli/src/cli_util.rs

```diff
@@ -598,26 +614,26 @@ impl WorkspaceCommandHelper {
         repo: Arc<ReadonlyRepo>,
         loaded_at_head: bool,
     ) -> Result<Self, CommandError> {
-        let settings = command.settings.clone();
+        let settings = command.data.settings.clone();
         let commit_summary_template_text =
             settings.config().get_string("templates.commit_summary")?;
-        let revset_aliases_map = revset_util::load_revset_aliases(ui, &command.layered_configs)?;
+        let revset_aliases_map =
+            revset_util::load_revset_aliases(ui, &command.data.layered_configs)?;
         let template_aliases_map = command.load_template_aliases(ui)?;
-        let may_update_working_copy = loaded_at_head && !command.global_args.ignore_working_copy;
+        let may_update_working_copy =
+            loaded_at_head && !command.data.global_args.ignore_working_copy;
         let working_copy_shared_with_git = is_colocated_git_workspace(&workspace, &repo);
         let path_converter = RepoPathUiConverter::Fs {
-            cwd: command.cwd.clone(),
+            cwd: command.data.cwd.clone(),
             base: workspace.workspace_root().clone(),
         };
         let helper = Self {
-            string_args: command.string_args.clone(),
-            global_args: command.global_args.clone(),
-            settings,
+            command: command.clone(),
             workspace,
             user_repo: ReadonlyUserRepo::new(repo),
-            revset_extensions: command.revset_extensions.clone(),
+            revset_extensions: command.data.revset_extensions.clone(),
             commit_summary_template_text,
-            commit_template_extensions: command.commit_template_extensions.clone(),
+            commit_template_extensions: command.data.commit_template_extensions.clone(),
```

Comment:
nit: `revset_extension` and `commit_template_extensions` can be removed, too.

Reply:
Done